client-js/app/elements/labrad-server.ts

```diff
@@ -0,0 +1,9 @@
+import {ServerInfo, SettingInfo} from '../scripts/manager';
```

Comment:
This element shows information about a server (it's doc strings and setting definitions). It's been there for a long time, I just refactored it to put the code in typescript, as with other elements.